nncf/torch/statistics/aggregator.py

```diff
@@ -0,0 +1,63 @@
+
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Dict
+
+import torch
+import numpy as np
+
+from nncf.common.graph.transformations.commands import TransformationPriority
+from nncf.common.graph.transformations.layout import TransformationLayout
+from nncf.common.tensor_statistics.aggregator import StatisticPointsContainer
+from nncf.common.tensor_statistics.aggregator import StatisticsAggregator
+from nncf.torch.tensor import PTNNCFTensor
+from nncf.torch.nncf_network import NNCFNetwork
+from nncf.torch.graph.transformations.commands import PTInsertionCommand
+
+
+
+class PTStatisticsAggregator(StatisticsAggregator):
+
+    def collect_statistics(self, model: NNCFNetwork) -> None:
+        with torch.no_grad():
+            with model.temporary_clean_view() as intermediate_model:
+                super().collect_statistics(intermediate_model)
+
+    def _register_statistics(self,
+                             outputs: Dict[str, PTNNCFTensor],
+                             statistic_points: StatisticPointsContainer) -> None:
+        return
```

Comment:
Why it is empty?

Reply:
Torch doesn't use output to register statistics, correspondent values are being registering during pre/post hook calls